solr/core/src/test/org/apache/solr/response/TestRawTransformer.java

```diff
@@ -60,7 +61,11 @@ public static void beforeClass() throws Exception {
     if (random().nextBoolean()) {
       initStandalone();
       JSR.start();
-      CLIENT = JSR.newClient();
+      if (random().nextBoolean()) {
+        CLIENT = JSR.newClient();
+      } else {
+        CLIENT = new EmbeddedSolrServer(JSR.getCoreContainer(), null);
```

Comment:
it works; I re-rean the test enough times to see it does, with code coverage.  Not in this PR, I also played with flipping SolrTestCaseHS and some other ZK endpoint test, which helped improve this PR by uncovering things, but those changes seemed like too much scope right now.